pkg/daemon/update.go

```diff
@@ -1582,7 +1592,8 @@ func createOrigFile(fromPath, fpath string) error {
 		// took over so it can just remove it when deleting stale data, as opposed as restoring a file
 		// that was shipped _with_ the underlying OS (e.g. a default chrony config).
 		if err := os.MkdirAll(filepath.Dir(noOrigFileStampName(fpath)), 0755); err != nil {
-			return errors.Wrapf(err, "creating no orig parent dir: %v", err)
+			errs := kubeErrs.NewAggregate([]error{err, err})
```

Comment:
Same as above. However, in this case, you can aggregate the error from `os.Stat`.